test/ruleLoaderTests.ts

```diff
@@ -18,6 +18,7 @@ import * as Lint from "./lint";
 describe("Rule Loader", () => {
     const path = require("path");
     const rulesDirectory = path.join(global.process.cwd(), "build/rules");
+    const rulesDirectories = [rulesDirectory];
```

Comment:
Since I couldn't understand what needs to be done here, didn't change it in https://github.com/palantir/tslint/commit/ee60a0cd1b858bd9555f21e9b4efdb501ebb570d.